development/build/display.js

```diff
@@ -0,0 +1,156 @@
+const randomColor = require('randomcolor')
+const chalk = require('chalk')
+
+module.exports = { setupTaskDisplay, displayChart }
+
+const SYMBOLS = {
+  Empty: '',
+  LeftCorner: '◺',
```

Comment:
Nit: these corners and the `EOL` character aren't used

Reply:
pardon my copy pasta :spaghetti: